src/Repository/DefaultSluggableRepository.php

```diff
@@ -40,4 +41,12 @@ public function isSlugUniqueFor(SluggableInterface $sluggable, string $uniqueSlu
         return ! (bool) $queryBuilder->getQuery()
             ->getSingleScalarResult();
     }
+
+    public function isSlugUnique(
+        string $uniqueSlug,
+        SluggableInterface $newOrUpdated,
+        SluggableInterface $exisiting
+    ): bool {
+        return $uniqueSlug !== $exisiting->getSlug();
+    }
```

Comment:
The name `isSlugUnique` is suggesting there can be conflicting slugs, which sound like conflicting ids.

What does it actually do?

Reply:
The method in the default-repository is just the extracted comparison of slugs from the [SluggableEventSubscriber::isSlugUniqueInUnitOfWork](https://github.com/KnpLabs/DoctrineBehaviors/blob/master/src/EventSubscriber/SluggableEventSubscriber.php#L116) method.

It checks, wether the slugs of two entities collide. I moved the check to the repository, because different entities can have different uniqueness constraints. Like, the default one with no constraints at all or my example with the pages per company.

I realize that the naming is bad, because it's just the comparison of two entities, and not the whole set of entities.